src/planscape/conditions/raster_utils.py

```diff
@@ -127,3 +172,26 @@ def fetch_or_compute_condition_stats(
             count=stats['count'])
 
     return condition_stats
+
+
+# Fetches raw raster pixel values for all non-NaN pixels that intersect with
+# geo.
+def get_condition_values_from_raster(
+        geo: GEOSGeometry, raster_name: str) -> ConditionPixelValues:
```

Comment:
nit - depending on who the caller is, this might be a funky string to expose. we could potentially try to do the db mapping of condition<->raster_name in the backend. not a blocker

Reply:
This is just a utility function - I don't intend for it to be exposed directly to users as an API (maybe I'm misunderstanding the comment).